vtki/plotting.py

```diff
@@ -1471,6 +1482,19 @@ def add_scalar_bar(self, title=None, n_labels=5, italic=False, bold=True,
 
             self._scalar_bar_actors[title] = self.scalar_bar
 
+        if interactive and hasattr(self, 'iren'):
+            self.scalar_widget = vtk.vtkScalarBarWidget()
```

Comment:
@akaszynski, this isn't working with the Qt plotting windows. Do you have any idea why this might be?

This works fine with the `Plotter` class but it is not interactive with the `BackgroundPlotter`. Note that the interactive axes widget works fine for the `BackgroundPlotter`.

Reply:
This is working. See hard to reproduce bug in #97